tests/sentry/receivers/test_onboarding.py

```diff
@@ -482,3 +485,155 @@ def test_onboarding_complete(self):
+                                            "      ignoreNextOnError();",
+                                            "",
+                                            "      withScope((scope) => {",
+                                            "        scope.addEventProcessor((event) => {",
+                                        ],
+                                        "in_app": False,
+                                        "data": {
+                                            "sourcemap": "https://s1.sentry-cdn.com/_static/dist/sentry/sourcemaps/vendors-node_modules_emotion_is-prop-valid_node_modules_emotion_memoize_dist_memoize_brows..."
+                                        },
+                                    }
+                                ]
+                            },
+                            "raw_stacktrace": {
+                                "frames": [
+                                    {
+                                        "function": "o",
+                                        "filename": "/_static/dist/sentry/chunks/vendors-node_modules_emotion_is-prop-valid_node_modules_emotion_memoize_dist_memoize_browser_-4fe4bd.255071ceadabfb67483c.js",
+                                        "abs_path": "https://s1.sentry-cdn.com/_static/dist/sentry/chunks/vendors-node_modules_emotion_is-prop-valid_node_modules_emotion_memoize_dist_memoize_browser_-4fe4bd.255071ceadabfb67483c.js",
+                                        "lineno": 2,
+                                        "colno": 37098,
+                                        "pre_context": [
+                                            "/*! For license information please see vendors-node_modules_emotion_is-prop-valid_node_modules_emotion_memoize_dist_memoize_browser_-4fe4bd. {snip}"
+                                        ],
+                                        "context_line": "{snip} .apply(this,arguments);const i=o.map((e=>c(e,t)));return e.apply(this,i)}catch(e){throw l(),(0,i.$e)((n=>{n.addEventProcessor((e=>(t.mechani {snip}",
+                                        "post_context": [
+                                            "//# sourceMappingURL=../sourcemaps/vendors-node_modules_emotion_is-prop-valid_node_modules_emotion_memoize_dist_memoize_browser_-4fe4bd.fe32 {snip}"
+                                        ],
+                                        "in_app": False,
+                                    },
+                                ],
+                            },
+                        }
+                    ]
+                },
+            },
+        )
+
+        record_analytics.assert_called_with(
+            "first_event_with_minified_stack_trace_for_project.sent",
+            user_id=self.user.id,
+            organization_id=project.organization_id,
+            project_id=project.id,
+            platform="javascript",
+            url="http://localhost:3000",
+        )
```

Comment:
We should also test that after logging the first event no further events are logged when receiving future errors from the project.